src/Traits/Http.php

```diff
@@ -227,7 +227,11 @@ protected function replyMarkupToString(array $params): array
     protected function uploadFile(string $endpoint, array $params, $inputFileField): TelegramResponse
     {
         //Check if the field in the $params array (that is being used to send the relative file), is a file id.
-        if (!isset($params[$inputFileField]) || $this->hasFileId($inputFileField, $params)) {
+        if (!isset($params[$inputFileField])) {
```

Comment:
This is broken.

Reply:
I didn't test since this PR was merged. Please create a PR with a fix if you found the bug? Thanks.